### ISSUE TITLE
Request to add a detailed usage guide for people who are not familiar with python and the terminal

### ISSUE THREAD

**Issue by Lsatric** [kichiki/finsta360#3]
Hi! I have corrupted insta 360videofiles, unfortanly I don't know who to use your code. I have never used pyton and don't know if I should launch a file or open the terminal. So please add a step by step example where you show the steps from scratch with some screenshots, or by a detailed script.
That would be really helpful. 
Thanks

**Comment by caronelorenzo** [comment]
Yes, it would be fantastic. Hope someone could write a detailed guide.
